Updated german translations

garritfra opened the pull request:
The german translations are a bit sloppy. I'll do my best to look over as many exercises as possible in the future.

<!-- freeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/Contributors -->
<!-- Make sure that your PR is not a duplicate -->

#### Pre-Submission Checklist
<!-- Go over all points below, and after creating the PR, tick all the checkboxes that apply. -->
<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
<!-- What type of change does your code introduce? After creating the PR, tick the checkboxes that apply. -->
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)

#### Checklist:
<!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask in the Contributors room linked above. We're here to help! -->
- [ ] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description
<!-- Describe your changes in detail -->

garritfra committed:
Updated german translations 

The german translations are a bit sloppy. I'll do my best to look over as many exercises as possible in the future.

garritfra committed:
basic-html-and-html5.json translations

Updated german translations for basic-html-and-html5.json

garritfra committed:
Merge pull request #1 from garritfra/staging

basic-html-and-html5.json translations

raisedadead commented on seed/challenges/00-getting-started/getting-started.json:
```diff
@@ -97,26 +97,26 @@
               "//i.imgur.com/6ibIavQ.jpg",
-              "Ein Bild von den 4 Vorteilen von freeCodeCamp: Vernetz Dich, Lerne Javascript, Bau ein Portfolio auf, Hilf Non-Profit-Organisationen",
-              "Willkommen bei freeCodeCamp. Wir sind eine Open Source Gemeinschaft von motivierten Leuten, die programmieren lernen und Non-Profit-Organisationen helfen",
+              "Ein Bild von den 4 Vorteilen von freeCodeCamp: Vernetz Dich, Lerne Javascript, bau dein eigenes Portfolio, hilf aktiv bei Non-Profit-Organisationen,
```
you are missing a `"` at the end of this line

raisedadead reviewed (dismissed):
There is a missing `"` at the end of this line.
Once you have made that commit, amend your commit and squash them.

If you need assistance, please reachout to the contributors chat room.

You can find all details also in the contributing guide.

garritfra committed:
Update getting-started.json

raisedadead commented:
@garritfra thanks, can you find some one to peer review this? We will merge in one week from now.

garritfra commented:
I don't know any german contributers to review this issue. Is there a way to find someone?
I currently don't have access to a git console to squash my commits. I'll do that in a few hours.

raisedadead commented:
Yup, you should post on the forum, the chat room with link to this pull request. And usually people who are interested will get back to you. Thanks again for helping us in translation efforts.

garritfra commented:
@Knochenmark will QA it

Knochenmark commented on seed/challenges/00-getting-started/getting-started.json:
```diff
@@ -128,13 +128,13 @@
               "//i.imgur.com/UVB9hxp.jpg",
               "Ein Foto von einem Camper in einem Cafe der an freeCodeCamp Projekten arbeitet.",
-              "Wir glauben dass Menschen am Besten beim Tun lernen. Deswegen wirst du die meiste Zeit damit beschäftigt sein an tatsächlichen Projekten zu arbeiten. Wir werden dir eine Liste an Anforderungen (agile User Stories) geben und du wirst dich damit auseinander setzen diese zu erfüllen.",
+              "Wir glauben, dass Menschen am Besten durch aktives Anwenden lernen. Deswegen wirst du die meiste Zeit damit beschäftigt sein an tatsächlichen Projekten zu arbeiten. Wir werden dir eine Liste an Anforderungen (agile User Stories) geben. Deine Aufgabe ist es, diese zu erfüllen.",
```
How about this?
` Dazu geben wir dir eine Liste an Anforderungen (agile User Stories) und deine Aufgabe ist es, diese umzusetzen.",`

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -443,7 +443,7 @@
         "de": {
           "description": [
-            "Denk daran, dass du einen Kommentar mit <code>&#60;!--</code> anfangen und mit <code>--&#62;</code> wieder beenden werden.",
+            "Denk daran, dass du einen Kommentar mit <code>&#60;!--</code> anfangen und mit <code>--&#62;</code> wieder beendest.",
```
"dass ein Kommentar mit ... beginnt und mit ... endet" Maybe something along the lines?

garritfra commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -80,15 +80,15 @@
             "Schließende Tags so:",
             "<code>&#60;/h1&#62;</code>",
-            "Beachte, dass der einzige Unterschied zwischen öffnenden und schließenden Tags ist, das schließende Tags ein Slash nach der sich öffnenden spitzen Klammer folgt.",
+            "Beachte, dass der einzige Unterschied zwischen öffnenden und schließenden Tags ist, dass schließende Tags ein Slash nach der sich öffnenden spitzen Klammer folgt.",
```
update translation!
link PR #15995

garritfra replied:
should be "...dass bei"

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -1262,10 +1262,10 @@
       "translations": {
         "de": {
-          "title": "Erstelle ein Textfeld",
+          "title": "Ein Textfeld erstellen",
```
you shouldnt change this, it would be inconsistent with the rest of the titles

garritfra reviewed (commented):
translation error slipped through my review ;)

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -1262,10 +1262,10 @@
           "description": [
             "Nun erstellen wir ein Web Formular.",
-            "Eingabefelder sind eine bewährte Möglichkeit um Daten von Usern zu erhalten.",
+            "Eingabefelder sind eine beliebte Möglichkeit um Daten von Usern zu erhalten.",
```
this changes the meaning, "bewährte" means it is reliable and therefore should be used, while "beliebte" indicates a preference

Knochenmark reviewed (changes requested):
Ive added some remarks. There are a few minor things that might make the sentence sound better. There are also a few things you shouldnt change though, since they either introduce inconsistency or change the meaning of the sentence.

raisedadead commented:
@Knochenmark awesome thanks for the review!

QuincyLarson commented:
@garritfra Thanks for your contributions! 

Please take a look at @Knochenmark's suggestions. I'm not a German speaker, so use your best judgement here. Then when you're ready, please comment here and we can merge this.

garritfra committed:
changed bau to erstelle

garritfra committed:
changed Studien to Studenten

garritfra committed:
changed "programmieren" to Programmieren

garritfra committed:
changed sloppy sentence

"Dazu geben wir dir eine Liste an Anforderungen (agile User Stories) und deine Aufgabe ist es, diese umzusetzen."

garritfra committed:
changed "dass" to "dass bei"

still not a perfect sentence but it works for now

garritfra committed:
updated sloppy sentence

"dass ein Kommentar mit ... beginnt und mit ... endet" Maybe something along the lines?

garritfra committed:
reset "ein Textfeld erstellen" to "erstelle ein Textfeld"

garritfra reviewed (commented):
Updated all requests. this is my first attempt in translating code, so i knew there would be some mistakes. Thanks @Knochenmark !

garritfra commented:
btw, is there a way for you guys to squash the commits? I currently can't access a git terminal to do so myself

raisedadead commented:
@garritfra no problem, we will do that for you. @Knochenmark can you take one last look and approve, before we merge?

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -374,7 +374,7 @@
         "de": {
           "description": [
-            "Kommentieren erlaubt dir Kommentare innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
+            "Kommentieren erlaubt es dir, Bemerkungen innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
```
I wouldnt change **Kommentare** here, but keep the other change. Only revert the translation into Bermerkungen.

comments have a specific meaning in this context, that can differ from _Bemerkungen_. Comments can also include documentation for example.

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -374,7 +374,7 @@
           "description": [
-            "Kommentieren erlaubt dir Kommentare innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
+            "Kommentieren erlaubt es dir, Bemerkungen innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
             "Dies ist auch ein guter Weg, um Code inaktiv zu machen ohne diesen komplett löschen zu müssen.",
```
um Code `zu deaktivieren` ohne diesen...

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -374,7 +374,7 @@
-            "Kommentieren erlaubt dir Kommentare innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
+            "Kommentieren erlaubt es dir, Bemerkungen innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
             "Dies ist auch ein guter Weg, um Code inaktiv zu machen ohne diesen komplett löschen zu müssen.",
             "Du kannst einen Kommentar mit <code>&#60;!--</code> starten und ihn mit <code>--&#62;</code> wieder beenden.",
```
change starten to beginnen

since we used that terminology in the next section. Should be more consistent.

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -708,13 +708,13 @@
-            "<code>a</code> Elemente, auch bekannt als <code>anchor</code> (Anker) Elemente, werden verwendet um auf Inhalte außerhalb der aktuellen Seite zu verlinken.",
+            "<code>a</code> Elemente, auch bekannt als <code>anchor</code> (Anker) Elemente, werden verwendet um auf Inhalte außerhalb der aktuellen Seite zu verweisen.",
             "So sieht ein <code>a</code> Element aus. In diesem Fall wird es innerhalb eines Paragraphen Elements verwendet. Das bedeutet dein Link wird innerhalb des Satzes erscheinen.",
             "<a href=\"//i.imgur.com/hviuZwe.png\" target=\"_blank\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"Eine Darstellung wie man Anker Elemente geschrieben werden mit dem gleichen Text wie in der folgenden Zeile\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
```
the alt text is wrong. maybe check the english translation if you want to fix this

garritfra replied:
What do you suggest as a translation?
I can't seem to find the original alt attribute

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -1265,7 +1265,7 @@
           "description": [
             "Nun erstellen wir ein Web Formular.",
-            "Eingabefelder sind eine bewährte Möglichkeit um Daten von Usern zu erhalten.",
+            "Eingabefelder sind eine beliebte Möglichkeit um Daten von Usern zu erhalten.",
```
revert this part to **bewährte**

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -1522,7 +1522,7 @@
           "title": "Füge eine Button zum Senden hinzu",
```
Füge eine**n** Button zum Senden hinzu

Knochenmark commented on seed/challenges/01-responsive-web-design/basic-html-and-html5.json:
```diff
@@ -1906,10 +1906,10 @@
           "title": "Standardmäßig ausgewählte Radiobuttons und Checkboxen",
           "description": [
             "Du kannst Checkboxen und Radiobuttons standardmäßig markiert machen, indem du das <code>checked</code> Attribut verwendest.",
```
markiert machen => markieren

Knochenmark reviewed (changes requested):
I found a few other things, you didnt came across.
It seems like you didnt reverted that one change regarding "beliebte" to "bewährte". I think you shouldnt touch that, it changes context slightly.

QuincyLarson commented:
@garritfra what do you think of @Knochenmark's recommended changes? Can you make the changes you agree with, then let us know when it's ready to be merged?

garritfra committed:
fixed "Tausenden" and "unbesetzten"

garritfra committed:
an --> mit

garritfra committed:
Reverted Bemerkungen to Kommentare

garritfra committed:
reverted beliebte to bewährte

garritfra committed:
eine --> einen

garritfra committed:
markiert machen --> markieren

garritfra reviewed (commented):
updated all requested translations except one. need suggestion for translation because i can't seem to find the original text lol

raisedadead commented:
@garritfra You could check the original translation by diff against staging.

Here is what I found, hope it helps:

https://github.com/freeCodeCamp/freeCodeCamp/blob/89306fb875382a5aa40251b04df073dd0cbc33ef/seed/challenges/01-responsive-web-design/basic-html-and-html5.json#L708-L719

garritfra commented:
I'm missing the english translation to the wrongly translated alt tag though

raisedadead commented:
Isn't that a little above in the link that I shared? 

https://github.com/freeCodeCamp/freeCodeCamp/blob/89306fb875382a5aa40251b04df073dd0cbc33ef/seed/challenges/01-responsive-web-design/basic-html-and-html5.json#L682

garritfra committed:
fixed anchor tag

garritfra reviewed (commented):
My bad! I simply skimmed over the code and didn't find it

Knochenmark commented:
👍
